{
  "uid" : "17a64ddf28afde92",
  "name" : "\"before all\" hook: set implementations for \"has an owner\"",
  "fullName" : "Contract: ProxyAdmin.\"before all\" hook: set implementations for \"has an owner\"",
  "historyId" : "a114cd4570e2bc344e3e89fa7a1edd22",
  "time" : {
    "start" : 1635554978482,
    "stop" : 1635554978482,
    "duration" : 0
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7bf10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7bf10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/proxy/transparent/ProxyAdmin.test.js:14:45)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ProxyAdmin"
  }, {
    "name" : "testClass",
    "value" : "Contract: ProxyAdmin"
  }, {
    "name" : "testMethod",
    "value" : "\"before all\" hook: set implementations for \"has an owner\""
  }, {
    "name" : "package",
    "value" : "Contract: ProxyAdmin"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "f6c994ecffd23012",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0435cd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554908810,
        "stop" : 1635554908810,
        "duration" : 0
      }
    }, {
      "uid" : "20f7aa7fb1bb530f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0435790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554904870,
        "stop" : 1635554904870,
        "duration" : 0
      }
    }, {
      "uid" : "2d9dbff5166771f8",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042e820>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554893993,
        "stop" : 1635554893993,
        "duration" : 0
      }
    }, {
      "uid" : "70f17f637afcc44e",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0682100>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554889431,
        "stop" : 1635554889431,
        "duration" : 0
      }
    }, {
      "uid" : "f7362ea31b247936",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0678ca0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554842241,
        "stop" : 1635554842241,
        "duration" : 0
      }
    }, {
      "uid" : "6a3b8311db4af98",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0670190>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554824435,
        "stop" : 1635554824435,
        "duration" : 0
      }
    }, {
      "uid" : "b891b7ff44d1d126",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0676100>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554811787,
        "stop" : 1635554811787,
        "duration" : 0
      }
    }, {
      "uid" : "e9ec90b7f1120b95",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0510e80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554809326,
        "stop" : 1635554809326,
        "duration" : 0
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "e27a0f8712fc1596",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/e27a0f8712fc1596",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551253648,
          "stop" : 1635551253648,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "17a64ddf28afde92.json",
  "parameterValues" : [ ]
}